### PR TITLE
fix(qaonlytimedistreport): add filter out qa user

### PR DESCRIPTION
**Pull request by aleksbykov**
Remove from report instances with owner qa

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~New configuration option are added and documented (in `sdcm/sct_config.py`)~
- [ ] ~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~
- [ ] ~All new and existing unit tests passed (CI)~
- [ ] ~I have updated the Readme/doc folder accordingly (if needed)~

### CODE REVIEW

File: sdcm/utils/cloud_monitor/report.py [hunk 1]
```diff
@@ -170,3 +170,9 @@ def _is_older_than_5days(create_time):
     @staticmethod
     def _is_older_than_7days(create_time):
         return create_time < pytz.utc.localize(datetime.utcnow() - timedelta(days=7))
+
+    def _is_user_be_skipped(self, instance):
+        return instance.owner == "qa" or instance.owner not in self.qa_users or instance.state != "running"
```
Comment (bentsi) [inline sdcm/utils/cloud_monitor/report.py:175]: `instance.state != "running"` is checked in `_is_instance_be_skipped`

File: sdcm/utils/cloud_monitor/templates/per_qa_user.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-    <h2>Instances</h2>
+    <h2>Instances are running</h2>
```
Comment (bentsi) [inline sdcm/utils/cloud_monitor/templates/per_qa_user.html:1]: `Running instances` or `Instances that are running`

**Review by bentsi — CHANGES_REQUESTED** [review]
address comments

**Comment by aleksbykov** [comment]
@bentsi , i addressed comments

File: sdcm/utils/cloud_monitor/report.py (hunk 1 above)
Comment (aleksbykov) [reply sdcm/utils/cloud_monitor/report.py]: fixed

File: sdcm/utils/cloud_monitor/templates/per_qa_user.html (hunk 2 above)
Comment (aleksbykov) [reply sdcm/utils/cloud_monitor/templates/per_qa_user.html]: fixed

File: sdcm/utils/cloud_monitor/report.py [hunk 3]
```diff
@@ -170,3 +170,9 @@ def _is_older_than_5days(create_time):
     @staticmethod
     def _is_older_than_7days(create_time):
         return create_time < pytz.utc.localize(datetime.utcnow() - timedelta(days=7))
+
+    def _is_user_be_skipped(self, instance):
+        return instance.owner == "qa" or instance.owner not in self.qa_users
```
Comment (juliayakovlev) [inline sdcm/utils/cloud_monitor/report.py:175]: Maybe it's worthy to do non case-sensitive check

**Review by juliayakovlev — COMMENTED** [review]
one small comment

File: sdcm/utils/cloud_monitor/report.py (hunk 3 above)
Comment (bentsi) [reply sdcm/utils/cloud_monitor/report.py:175]: It is done in the CloudInstance class
image

**Review by bentsi — APPROVED** [review]

**Pull request merged**